app/views/posts/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<h2>New Post</h2>
+
+<%= form_for @posts do |f| %>
```

Comment:
Again, would be very careful with how you name variables. You are creating a single new posts, not multiple ones (as `@posts` would imply).